lightning/src/routing/router.rs

```diff
@@ -1101,7 +1101,9 @@ where L::Target: Logger {
 		} }
 	}
 
-	let empty_node_features = NodeFeatures::empty();
+	let mut variable_length_onion_optional_features = NodeFeatures::empty();
+	variable_length_onion_optional_features.set_variable_length_onion_optional();
```

Comment:
```suggestion
	let mut onion_optional_features = NodeFeatures::empty();
	onion_optional_features.set_variable_length_onion_optional();
```

I'm a super fan of the explicative variable name, but I think that here the variable is too long? Mabey `onion_optional_features` is better?